app/lib/ga-log-adapter.js

```diff
@@ -0,0 +1,48 @@
+const dimensionMap = {
+  experiment: 1,
+  cohort: 2,
+  projectToken: 3,
+  userID: 4,
+  subjectID: 5,
+  workflowID: 6
```

Comment:
Does this need the extra `data` key?

Reply:
I just added the key while we were talking yesterday to show how easy it was--it's not actually used yet :)